react-native-libraries.json

```
@@ -13330,5 +13330,18 @@
     "ios": true,
     "android": true,
     "newArchitecture": false
+  },
+  {
+    "githubUrl": "https://github.com/reduxjs/redux-toolkit/tree/HEAD/packages/toolkit",
```

Hey @aryaemami59, thanks for adding new entry to the directory! 👍 

```suggestion
    "githubUrl": "https://github.com/reduxjs/redux-toolkit/tree/master/packages/toolkit",
```

Let's use main branch, instead of `HEAD` tag.